### ISSUE TITLE
LOOT taking way too long

### ISSUE THREAD

**Issue by Budroi** [loot/loot#436]
I'm not sure if its because Loot is being ran through Mod Organizer or because i have 252 mods.
When I run loot the screen pops up but it is empty and solid white (screen 1). It has been sitting at white screen for over ten minutes. task manager says its responding so I know its not froze.
![screen 1](https://cloud.githubusercontent.com/assets/10514699/7778949/c39b2dd8-0095-11e5-8856-6edc42548349.png)
 When i minimize and then open it back up the white is replaced with solid black (screen 2).

![screen 2](https://cloud.githubusercontent.com/assets/10514699/7778950/c3a16784-0095-11e5-8c37-a241af85fe0a.png)

Am I just supposed to wait for it to do its work or is something not working correctly because I have a really fast computer and there is not a single program (including the skyrim creation kit) that takes this long to do anything.

**Comment by Budroi** [comment]
For the time being i will be going back to the previous version which loaded in less than 5 seconds until a suitable answer\fix can be obtained.

**Comment by WrinklyNinja** [comment]
Have you tried [the suggested fix](https://github.com/loot/loot.github.io/wiki/LOOT-FAQs#loots-window-is-blank-when-im-starting-it-through-mod-organizer-whats-up) in the FAQs?

**Comment by Budroi** [comment]
Son of a...
How did I miss that. I will test that.

**Comment by Budroi** [comment]
That did not work.

*fix option 1 - Not applicable - I'm using the most recent version of MO - 1.3.5
*fix option 2 - Did not help

Also I figured out how I missed that.
I originally read your response when not at my pc so i clicked link to faqs in the message.
When i went to my PC and attempted to find the faqs page it took 10 min before I gave up and clicked the link in the message.
It seems that i am not proficient with navigating Github.

**Comment by WrinklyNinja** [comment]
> *fix option 2 - Did not help

Very odd. In that case, can you please [supply your debug logs](https://github.com/loot/loot.github.io/wiki/How-To-Report-Issues-Helpfully)?

**Comment by Budroi** [comment]
Okay I am not sure what happened.
I had downgraded after running the above tests.
Well I reinstalled now to get you those debug logs and I fired LOOT up outside of MO so that it would only open my default skyrim lists.
It did and I went in and turned debug on and set LOOT to NOT autodetect games (it had already found skyrim though) and I manually set it for skyrim.
Well when MO loaded it this time (it still has that argument set) it loaded quickly (very quickly).
I am going to try again without the argument and see what occurs, but it seems that my issue resolved itself..

**Comment by Budroi** [comment]
Yep works without the argument as well now.
Not sure what that problem was but the only difference between now and then was that I turned debug on and stopped LOOT from auto-detecting Skyrim and manually set it.

**Comment by WrinklyNinja** [comment]
> Sorry to have wasted your time with this.

It's only a waste of time if the problem doesn't actually get fixed: I'm glad you found a workaround, but it should just work with the game autodetection on. That said, it might be a bug in MO, since you say it works fine when you run it outside MO.

Could you please run LOOT through MO with the debug logging turned off and the game autodetection on, and when it gets to the blank screen, close it, then upload your debug logs for me? They should provide enough information for me to track down the cause of this issue, if it is an issue with LOOT and not MO.

**Comment by Budroi** [comment]
OK now i am just plain lost and confused.
Turned auto-detect back on. turned off debug and no argument.
And it loaded just fine.
But now I have a thought about that let me test it quick.

**Comment by Budroi** [comment]
OK, so the problem is MO related. I need to close out MO for it to update the arguments when loading something through MO.
So all the previous failed times were ran during a single instance of MO.
After the previous failed tests i haven't had time to play so i had no started MO until today.
When i booted MO today it still had the argument in it so all the testing that allowed successful running of LOOT was using that argument.
Per my last comment my thought was correct.
I restarted MO with the argument off and had the blank screen.
Added back in argument and test = still blank
restart MO and test = functioning correctly.

So at the end of the day adding the argument did work but MO must have an issue that is not allowing arguments to update in real time and instead requires a restart.
